### PR TITLE
Feature/fro 83 browse

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- Allow for browsing
- Provide links to browse by country and category from the geography page
- Update slug to `/geographies`

### CODE REVIEW

File: frontend/src/pages/search/index.tsx [hunk 1]
```diff
@@ -184,21 +184,18 @@ const Search = () => {
...
-
-    // setShowPDF(false);
   };
 
   const getCurrentSortChoice = () => {
     const field = searchCriteria.sort_field;
     const order = searchCriteria.sort_order;
     if (field === null && order === "desc") {
+      if (browsing) return "date:desc";
       return "relevance";
```
Comment (joel-wright) [inline frontend/src/pages/search/index.tsx:199]: Just one question, do we also hide the option "Relevance" when no search term is provided?

**Review by joel-wright — APPROVED** [review]
Seems reasonable to me, just the one question about the sort order options presented.

File: frontend/src/pages/search/index.tsx [hunk 2]
```diff
@@ -337,19 +325,14 @@ const Search = () => {
                       <div className="w-full flex justify-center h-96">
                         <Loader />
                       </div>
-                    ) : noQuery ? (
-                      <p className="mt-4 font-bold text-red-500 h-96">
-                        {/* TODO: make text translatable */}
-                        Please enter some search terms.
```
[the remark this answers is not in the document]
Comment (PatFawbertMills) [reply frontend/src/pages/search/index.tsx:343]: Agreed

File: frontend/src/pages/search/index.tsx (hunk 1 above)
Comment (PatFawbertMills) [reply frontend/src/pages/search/index.tsx:199]: Yes good question - if you look in the `filters/Sort.tsx` component you can see I'm selecting the sort options based on whether we are in 'browsing' mode or not.

Comment (joel-wright) [reply frontend/src/pages/search/index.tsx:199]: cheers!

**Pull request merged**